### PR TITLE
Add a note to the restricted APIs

### PR DESCRIPTION
**Pull request by JorikMollie**
A couple of our APIs can only be used after persmission. Added a note to these APIs.

### CODE REVIEW

File: source/reference/reseller-api/endpoints/get-login-link.rst [hunk 1]
```diff
@@ -15,6 +15,9 @@ cases, the merchant who was already logged in will remain logged in.
 
 When the customer has not yet accepted Mollie's conditions they must first do so before being redirected.
 
+.. note::
+   The use of this API is restricted. Please contact our partner team at Partner@mollie.com if you want to use this API.
```
Comment (MollieRick) [inline source/reference/reseller-api/endpoints/get-login-link.rst]:
```suggestion
   The use of this API is restricted. Please contact our partner team at partner@mollie.com if you want to use this API.
```

File: source/reference/reseller-api/endpoints/set-fees.rst [hunk 2]
```diff
@@ -10,6 +10,9 @@ Set fees
 
 The method set-fees can be used to change a merchant's rates.
 
+.. note::
+  The use of this API is restricted. Please contact our partner team at Partner@mollie.com if you want to use this API.
```
Comment (MollieRick) [inline source/reference/reseller-api/endpoints/set-fees.rst]:
```suggestion
  The use of this API is restricted. Please contact our partner team at partner@mollie.com if you want to use this API.
```

File: source/reference/reseller-api/endpoints/get-login-link.rst (hunk 1 above)
Comment (ndijkstra) [reply source/reference/reseller-api/endpoints/get-login-link.rst]: It's partners@mollie.com 😃

Comment (JorikMollie) [reply source/reference/reseller-api/endpoints/get-login-link.rst]: I'll ask Partner(s) to tell me what the correct email is. The text was approved by them so I assumed it was correct ;)

Comment (MollieRick) [reply source/reference/reseller-api/endpoints/get-login-link.rst]: Both are existing.

**Pull request merged**